app/js/map/map.controller.js

```diff
@@ -0,0 +1,119 @@
+(function() {
+  'use strict';
+  angular.module('cityInfo.map').controller('MapController', MapController);
```

Comment:
It doesn't create a module, it just fetches it. Creating a module happens with syntax angular.module('modulename', []). Creating happens in map.module.js.